tools/apiview/parsers/js-api-parser/package.json

```diff
@@ -9,8 +9,9 @@
     "js-tokens": "^6.0.0"
   },
   "devDependencies": {
-    "ts-node": "^8.10.2",
-    "typescript": "^3.9.7"
+    "@types/node": "^12.20.55",
```

Comment:
nitpick: Could we go higher? to 14 for example?

Reply:
Done.